docs/guides/troubleshooting.md

```diff
@@ -32,7 +32,7 @@ You can supply an array of alternate sources. [webm][can-webm] and/or [ogv][can-
 Video.js 7+ supports HLS and MPEG-DASH as standard as it includes [http-streaming][http-streaming], which uses [Media Source Extensions][can-mse] to play these formats in modern browsers.
 If choosing a single format, HLS is more convenient as iOS and some Android browsers which do not support MSE do have native HLS support.
 
-HLS is also possible with Flash on IE 11 on Windows 7, which does not support MSE, if you add the [flashls source handler][flashls].
+HLS is not possible with Flash on IE 11 on Windows 7, which does not support MSE.
```

Comment:
probably good to keep this change but I've been very confused as to what the Win7IE11+Flash story is. Is it actually being removed?

Reply:
i thought flash has included code for a while to block itself from running after the EOL. maybe if you had an old enough version installed it will still work. windows 7 itself is EOL two weeks later, although that doesn't mean people that will stop using it.